internal/services/compute/shared_image_version_resource_test.go

```diff
@@ -285,6 +285,28 @@ func TestAccSharedImageVersion_requiresImport(t *testing.T) {
 	})
 }
 
+func TestAccSharedImageVersion_uefiSettings(t *testing.T) {
```

Comment:
since `uefi_settings` is updatable, could you also please add a testcase for update operation?

Reply:
Added an additional testcase.